src/utils/SignalingTypingHandler.js

```diff
@@ -124,29 +119,16 @@ SignalingTypingHandler.prototype = {
 			return
 		}
 
-		this._store.commit('setTyping', {
+		this._store.dispatch('setTyping', {
 			token: this._store.getters.getToken(),
 			sessionId: participant.nextcloudSessionId,
 			typing: data.type === 'startedTyping',
 		})
 	},
 
-	_handleParticipantsJoined(SignalingParticipantList, participants) {
-		if (!this._typing) {
-			return
-		}
-
-		for (const participant of participants) {
-			this._signaling.emit('message', {
-				type: 'startedTyping',
-				to: participant.signalingSessionId,
-			})
-		}
-	},
```

Comment:
It seems to me, that the problem here was only with `if (!typing)` condition.

The "this.typing" state in signaling is inconsistent with store state. This state should be only in the store state.